src/test/resources/api_query_response_no_projects_found.json

```diff
@@ -0,0 +1,10 @@
+{
+  "@context": "https://example.org/search-api-context.json",
+  "id": "https://api.dev.nva.aws.unit.no/project/search?QUERY_PARAMS",
+  "size": 0,
+  "searchString": "title=reindeer",
+  "processingTime": 1000,
+  "firstRecord": 0,
+  "nextResults": "",
```

Comment:
Next results is a URI, while URI creation from an empty string is valid, it creates a null object, which I think would be misleading from an interface perspective, delegating discovery of null objects to the consumer.